dist/war/getstarted/assets/css/custom.css

```diff
@@ -8,3 +10,203 @@ body {
     padding: 40px 15px;
     text-align: center;
 }
+
+.form-style-1 {
+    margin: 10px auto;
+    max-width: 400px;
+    padding: 2px 12px 10px 20px;
+    font: 18px "Lucida Sans Unicode", "Lucida Grande", sans-serif;
+}
+
+.form-style-1 li {
+    padding: 0;
+    display: block;
+    list-style: none;
+    margin: 10px 0 0 0;
+}
+
+.form-style-1 label {
+    margin: 0 0 0 0;
+    padding: 0 0 0 4px;
+    display: block;
+    font-weight: bold;
+    font-size: 12px;
+    text-align: left;
+}
+
+.form-style-1 input[type=text],
+.form-style-1 input[type=date],
+.form-style-1 input[type=datetime],
+.form-style-1 input[type=number],
+.form-style-1 input[type=search],
+.form-style-1 input[type=time],
+.form-style-1 input[type=url],
+.form-style-1 input[type=email],
+.form-style-1 input[type=password],
+.form-style-1 input[type=hidden]
+
```

Comment:
I guess you need a colon here?

Reply:
Actually, many resources were missing. I added them all.